blogposts/announcing-sourcegraph-3.1.md

```diff
@@ -0,0 +1,121 @@
+
+We're actively improving code intelligence in all languages, so please [send us feedback](https://github.com/sourcegraph/sourcegraph/issues/new?assignees=&labels=&template=feature_request.md&title=Language%20extension%20improvement%20for)!```
+
+## Site admin improvements
+
+We've added a couple features to help site admins manage a Sourcegraph instance in an enterprise environment:
+
+- Improved [management console](https://docs.sourcegraph.com/admin/management_console) JSON editor for critical configuration, now with editor completion, inline documentation, and validation.
+- Querying users by email address.
+- [Prevent users from changing their username](https://docs.sourcegraph.com/admin/config/critical_config#reference) (from the username set by SSO)
+
+Making configuration changes through the JSON editor is now easier too with easy to access documentation in the editor itself.
+
+<!-- TODO (ryan): GIF showing editor using the help tool tip with an item that requires a choice of values -->
+
+## Improved configuration management and documentation
+
+Sourcegraph 3.0 introduced the [management console](https://about.sourcegraph.com/blog/sourcegraph-3.0#management-console-and-site-config-improvements) for editing critical configuration to allow recovery from misconfiguration in the site admin.
+
+We've [updated our documentation](https://docs.sourcegraph.com/admin/config) to reflect this change, separating configuration into two sections:
+
+- [Critical configuration](https://docs.sourcegraph.com/admin/config/critical_config).
+- [Site configuration](https://docs.sourcegraph.com/admin/config/site_config).
+
+The documentation pages for [site configuration](https://docs.sourcegraph.com/admin/config/site_config#reference) and [critical configuration](https://docs.sourcegraph.com/admin/config/critical_config#reference) now describe each option and give examples, to make it easier for you to configure your Sourcegraph instance. You can get the same documentation and examples in the JSON editors for both types of configuration, too.
+
+## New Sourcegraph extension APIs
+
+The [Sourcegraph extension API](https://docs.sourcegraph.com/extensions) has two new APIs to make it easier to add interactive features to code views on Sourcegraph, GitHub, and other similar tools:
+
+- [API for accessing the current text selection range(s)](https://github.com/sourcegraph/sourcegraph/issues/702)
+- [APIs for listening to changes to the active text editor and its selections](https://github.com/sourcegraph/sourcegraph/issues/2026).
+
+This means the [Git extras Sourcegraph extension](https://sourcegraph.com/extensions?query=git-extras) works better than ever. Want this functionality on GitHub too? Just install the [Sourcegraph browser extension](https://docs.sourcegraph.com/integration/browser_extension).
+
+<!-- TODO (ryan): Gif of git-extras extension -->
+
+## Fixed release schedule: a new Sourcegraph release on the 20th of every month
```

Comment:
I think this should be at the top.

Reply:
I put it at the bottom because I wanted new features and improvements to be first.

If raising awareness is what you're after, what about a blog post about the change to our release process? Not just about the fixed monthly release date, but the release checklist as well.

Resolving for now.